doc/lsst.pipe.tasks/index.rst

```diff
@@ -12,3 +12,6 @@ Python API reference
 ====================
 
 .. automodapi:: lsst.pipe.tasks
+   :skip: AssembleCoaddTask, SafeClipAssembleCoaddTask, CompareWarpAssembleCoaddTask
```

Comment:
To get the API docs to show up, you'll need to change this to:

```
.. automodapi:: lsst.pipe.tasks.assembleCoadd
```

We'll generalize it to all of `lsst.pipe.tasks` when those APIs are available.

Reply:
Two questions here. (1) I was originally going to have just assembleCoadd listed, but it looks like @r-owen recently added all of lsst.pipe.tasks instead. Perhaps there's some special case I'm missing? (2) Does your comment here affect the `:skip:` line or just the `.. automodapi::` line?

Reply:
The comment is a replacement for both lines.

`lsst.pipe.tasks` doesn't import anything, turns out, so we need to point directly at individual modules